governance/org-docs/CHARTER.md

```diff
@@ -0,0 +1,61 @@
+# Charter for the [Altair-Viz] Organization
+
+This is the organizational charter for the Altair-Viz Organization (the "Organization"). By adding their name to the [Steering Committee.md file](./STEERING-COMMITTEE.md), Steering Committee members agree as follows.
+
+## 1. Mission
+
+Altair-Viz builds visualization tools with the main focus on Vega-Altair which is a declarative visualization library in Python. Its simple, friendly and consistent API, built on top of the powerful Vega-Lite grammar, empowers you to spend less time writing code and more time exploring your data.
+
+## 2. Steering Committee
+
+**2.1 Purpose**. The Steering Committee will be responsible for all technical oversight, project approval and oversight, policy oversight, and trademark management for the Organization.
+
+**2.2 Composition**. The Steering Committee voting members are listed in the steering-committee.md file in the repository.
+Voting members may be added or removed by no less than 3/4 affirmative vote of the Steering Committee.
+The Steering Committee will appoint a Chair responsible for organizing Steering Committee activity.
+
+## 3. Voting
+
+**3.1. Decision Making**. The Steering Committee will strive for all decisions to be made by consensus. While explicit agreement of the entire Steering Committee is preferred, it is not required for consensus. Rather, the Steering Committee will determine consensus based on their good faith consideration of a number of factors, including the dominant view of the Steering Committee and nature of support and objections. The Steering Committee will document evidence of consensus in accordance with these requirements. If consensus cannot be reached, the Steering Committee will make the decision by a vote.
+
+**3.2. Voting**. The Steering Committee Chair will call a vote with reasonable notice to the Steering Committee, setting out a discussion period and a separate voting period. Any discussion may be conducted in person or electronically by text, voice, or video. The discussion will be open to the public. In any vote, each voting representative will have one vote. Except as specifically noted elsewhere in this Charter, decisions by vote require a simple majority vote of all voting members.
```

Comment:
Good point, thanks for highlighting! I'm ok with it as I also understand that votes would be rather rare and only happen in case there does not seem to be a consensus among the committee members. What I don't quite understand is the phrasing `The Steering Committee will document evidence of consensus in accordance with these requirements.` What kind of decisions should we document? I'd interpret it that discussions in a pull request/issue/GH discussion serve as documentation. If we have a video call, we could post a slimmed down version of the minutes in a GitHub discussion outlining the decisions reached.